api/src/pcapi/core/chronicles/api.py

```diff
@@ -215,6 +216,35 @@ def save_chronicle(
         mark_transaction_as_invalid()
 
 
+def _get_offers(
+    product_identifier_type: models.ChronicleProductIdentifierType, product_identifier: str | None
+) -> list[offers_models.Offer]:
+    if not product_identifier:
+        return []
+
+    oldest_existing_chronicle_id = (
+        db.session.query(models.Chronicle.id)
+        .filter(
+            models.Chronicle.productIdentifierType == product_identifier_type,
+            models.Chronicle.productIdentifier == product_identifier,
+        )
+        .order_by(models.Chronicle.id.desc())
+        .scalar()
```

Comment:
Que se passe-t-il si on a plusieurs chroniques avec le même EAN ? De ce que je comprends de cette requête, il faudrait limiter le résultat à la première.
```suggestion
        .limit(1)
        .scalar()
```